src/main/java/anto/Parser.java

```diff
@@ -0,0 +1,206 @@
+package anto;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Parser class handles all reading of inputs from command line and decides what to do.
+ */
+public class Parser {
+
+    private Ui ui;
+    private TaskList taskList;
+    private Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Creates a Parser class.
+     * @param ui Ui class that handles printing to command line.
```

Comment:
Likewise, should there be an empty line here between description and parameter section?